inversion_specifications/NAME-RHIME/overview.md

```diff
@@ -0,0 +1,57 @@
+# RHIME inverse model specification
+
+Transport model and inversion setup for the Univeristy of Bristol Regional Hierarchical Inverse Modelling Environment (RHIME). The following applies to all species, unless specified in a species specific file in this folder.
+
+## Transport model
+
+NAME model (version 7.2) developped by the UK Met Office (Muller et al. 2013), a Langrangian particle model that we run backwards in time for each measurement site used in the inversion.
+
+### Scope
+
+- Domain: 10.73°N to 79.06°N and from 97.9°W to 39.38°E.
+- Period: 2018-01-01 to 2023-01-01
+
+### Meteorology
+
+The meteorological fields from the UK Met Office Unified Model are used. The model is run on a grid of horizontal resolution 0.0135° x 0.0135° over the UK (Manning and Redington et al. 2021). Vertically, it uses 57 levels, expanding to 12.2km.
```

Comment:
The values you've quoted are not consistent with those in Table 1 of that paper. You need to note that the horizontal resolution of the "global" met has changed with time.

Reply:
Also, should be referenced as Manning et al. (2021), no?

Reply:
I took the last line of table 1, which give the resolution for UK and Ireland. I realize that as we use whole Europe it's indeed global that matter, right? And we do our runs for 2018 to 2023, so the second-to-last should be quoted, right ? (July 2017 to present, horizontal : 0,141 x 0.0938 degrees, 59 vertical levels up to 29km)

For the ref, it said that the two first authors contributed equally so didn't really know how to handle that. But if you confirm I'll put Manning 2021.

Reply:
> I took the last line of table 1, which give the resolution for UK and Ireland. I realize that as we use whole Europe it's indeed global that matter, right? And we do our runs for 2018 to 2023, so the second-to-last should be quoted, right ? (July 2017 to present, horizontal : 0,141 x 0.0938 degrees, 59 vertical levels up to 29km)

Yes, that's right
 
> For the ref, it said that the two first authors contributed equally so didn't really know how to handle that. But if you confirm I'll put Manning 2021.

Ah, good point

Reply:
Sorry, I just realised that I didn't understand your question correctly. We embed the UKV meteorology within the global met fields, so we need to reference both